crates/ruff_linter/src/rule_redirects.rs

```diff
@@ -103,6 +103,16 @@ static REDIRECTS: Lazy<HashMap<&'static str, &'static str>> = Lazy::new(|| {
         ("TRY200", "B904"),
         ("PGH001", "S307"),
         ("PGH002", "G010"),
+        // flake8-trio and flake8-async merged with name flake8-async
+        ("TRIO", "ASYNC"),
```

Comment:
I think we should change `TRIO` to only map to `ASYNC1` because the old `ASYNC100` rules weren't part of the `ASYNC` selector. That's still not perfect because it will enable `ASYNC116` but it is closer to what users had before.